modules/graceful-shutdown.adoc

```diff
@@ -33,18 +33,69 @@ If your cluster fails to recover, follow the steps to restore to a previous clus
 
 .Procedure
 
-. If you are shutting the cluster down for an extended period, determine the date on which certificates expire.
+. If you plan to shut down the cluster for an extended period of time, determine the date that cluster certificates expire.
++
+You must restart the cluster prior to the date that certificates expire. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
+
+.. Check the expiration date for the `kube-apiserver-to-kubelet-signer` CA certificate:
 +
 [source,terminal]
 ----
-$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
+$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}{"\n"}'
 ----
 +
 .Example output
+[source,terminal]
 ----
 2022-08-05T14:37:50Zuser@user:~ $ <1>
 ----
-<1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
+<1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date.
+
+.. Check the expiration date for the kubelet certificates:
+
+... Start a debug session for a control plane node by running the following command:
```

Comment:
Can they check a single node, just any node? Or have to check multiple/specifically a control plane node or anything?

Reply:
Given the master nodes can be scaled up/down at any point, we should suggest to actually sync certificates of all nodes by rebooting all master nodes one by one before shutting down the cluster. Effectively making the expiration date almost identical everywhere. Otherwise, all master nodes should be checked and the minimum of all dates picked.

Reply:
Thanks, I can update to suggest to check all control plane nodes and use the earliest date

Reply:
@ingvagabund Updated for this, let me know what you think